paasta_tools/api/views/instance.py

```diff
@@ -101,10 +101,8 @@ def instance_status(request):
     instance = request.swagger_data.get('instance')
     verbose = request.matchdict.get('verbose', False)
 
-    instance_status = {}
-    instance_status['service'] = service
-    instance_status['instance'] = instance
-
+    instance_status = {'service': service,
+                       'instance': instance}
```

Comment:
I tend to prefer single-line or line-per-key/value (or line-per-argument for function calls) over visual-indent.

I usually use visual-indent only when a function call is getting too long, and it has many short arguments that would make argument-per-line have too many lines, or when other nearby lines are single-line, like with our use of mocks in tests.

We should probably write this down in the style guide...